examples/ansible/roles/alp_reboot/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Not sure need to include "empty" files like this defaults/main.yml or the following handlers/main.yml.

They are not required, and if you are not need/using this files, it is better not to include them.

Reply:
Removed unused ansible role files